cmd/ui-errors.go

```diff
@@ -0,0 +1,170 @@
+		"Invalid cache excludes value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXCLUDE: List of cache exclusion patterns delimited by `;`.",
+	)
+
+	uiErrInvalidCacheExpiryValue = newUIErrFn(
+		"Invalid cache expiry value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXPIRY: Cache expiry duration in days.",
+	)
+
+	uiErrInvalidCredentials = newUIErrFn(
+		"Passed credentials are not suitable for use",
+		"Please provide correct credentials",
+		`Access key length should be between minimum 3 characters in length.
+Secret key should not be between 8 and 40 characters.`,
+	)
+
+	uiErrInvalidErasureEndpoints = newUIErrFn(
+		"Invalid endpoints to use in erasure mode",
+		"Please provide correct combinations of local/remote paths",
+		"For more information, please refer to the following link: https://docs.minio.io/docs/minio-erasure-code-quickstart-guide",
+	)
+
+	uiErrInvalidNumberOfErasureEndpoints = newUIErrFn(
+		"The total number of endpoints is not suitable for erasure mode",
+		"Please provide an even number of endpoints greater or equal to 4",
+		"For more information, please refer to the following link: https://docs.minio.io/docs/minio-erasure-code-quickstart-guide",
+	)
+
+	uiErrStorageClassValue = newUIErrFn(
+		"Invalid storage class value",
+		"Please check the passed value",
+		`MINIO_STORAGE_CLASS_STANDARD: Format "EC:<Default_Parity_Standard_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in Standard mode. Object are stored in Standard mode, if storage class is not defined in Put request.
+MINIO_STORAGE_CLASS_RRS: Format "EC:<Default_Parity_Reduced_Redundancy_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in reduced redundancy mode. Objects are stored in Reduced Redundancy mode, if Put request specifies RRS storage class.
+Refer to the link https://github.com/minio/minio/tree/master/docs/erasure/storage-class for more information.`,
+	)
+
+	uiErrUnexpectedBackendVersion = newUIErrFn(
+		"Backend version seems to be too recent",
+		"Please update to the latest Minio version",
+		"",
+	)
+
+	uiErrInvalidAddressFlag = newUIErrFn(
+		"--address input is invalid",
+		"Please check --address parameter",
+		`--address binds a specific ADDRESS:PORT, ADDRESS can be an  IP or hostname (default:':9000')
+    Examples: --address ':443'
+              --address '172.16.34.31:9000'`,
+	)
+
+	uiErrInvalidFSEndpoint = newUIErrFn(
+		"The given endpoint is not suitable to activate standalone FS mode",
+		"Please check the given FS endpoint",
+		`FS mode requires only one writable disk path.
+Example 1:
+   $ minio server /data/minio/`,
+	)
+
+	uiErrUnableToWriteInBackend = newUIErrFn(
+		"Unable to write to the backend",
+		"Please ensure that Minio binary has write permissions to the backend",
+		"",
+	)
+
+	uiErrUnableToReadFromBackend = newUIErrFn(
+		"Unable to read from the backend",
+		"Please ensure that Minio binary has read permission from the backend",
```

Comment:
`"Please ensure that Minio binary has read permission from the backend"`
=>
`"Please ensure`~~`that`~~`Minio binary has read permission`**`s for`**`the backend"`